TEAMS.md

```diff
@@ -0,0 +1,64 @@
+# Teams
+
+## Core Team
+
+> The Core team is responsible for the direction of the project (roadmap), subteam leadership, the spec, and cross-cutting concerns.
+
+[@ekcasey][@ekcasey], [@hone][@hone], [@jkutner][@jkutner], [@nebhale][@nebhale], [@sclevine][@sclevine]
+
+## Implementation Team
+
+> The Implementation team is responsible for maintaining the components that constitute the Cloud Native Buildpacks reference implementation of the specification.
+
+### Maintainers
+
+[@ekcasey][@ekcasey], [@nebhale][@nebhale], [@sclevine][@sclevine]
+
+### Project Contributors
+
+[@ameyer-pivotal][@ameyer-pivotal], [@ashwin-venkatesh][@ashwin-venkatesh], [@dgageot][@dgageot], [@elbandito][@elbandito], [@matthewmcnew][@matthewmcnew], [@micahyoung][@micahyoung], [@murphybytes][@murphybytes], [@natalieparellano][@natalieparellano], [@simonjjones][@simonjjones], [@zmackie][@zmackie]
+
+## Platform Team
+
+> The Platform team is responsible for maintaining pack and any other platforms or platform components.
+
+### Maintainers
+
+[@ekcasey][@ekcasey], [@jromero][@jromero], [@sclevine][@sclevine]
+
+### Project Contributors
+
+[@ameyer-pivotal][@ameyer-pivotal], [@ashwin-venkatesh][@ashwin-venkatesh], [@dfreilich][@dfreilich], [@elbandito][@elbandito], [@matthewmcnew][@matthewmcnew], [@micahyoung][@micahyoung], [@murphybytes][@murphybytes], [@natalieparellano][@natalieparellano], [@simonjjones][@simonjjones], [@yaelharel][@yaelharel], [@zmackie][@zmackie]
+
+## Learning Team
+
+> The Learning team is responsible for maintaining the project’s documentation, website, and resources. 
+
+### Maintainers
+
+[@jkutner][@jkutner], [@jromero][@jromero]
+
+### Project Contributors
+
+[@ameyer-pivotal][@ameyer-pivotal], [@danielleadams][@danielleadams], [@dfreilich][@dfreilich], [@elbandito][@elbandito], [@murphybytes][@murphybytes], [@natalieparellano][@natalieparellano], [@simonjjones][@simonjjones], [@superhighfives][@superhighfives], [@zmackie][@zmackie]
+
```

Comment:
does this need to have a heading here?

Reply:
Do you mean for the items below this line? If so, they are just reference links to the profiles for the specific users. They don't show up in the markdown. They are used elsewhere throughout the document.